app/templates/components/doi-subjects.hbs

```diff
@@ -22,4 +22,4 @@
       {{/if}}
     {{/if}}
   </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
I am surprised that one of the linters does not complain about the removal of the newline at the end of the file (like Python ones do for example). This is just an observation/comment and not something I think needs to be changed before merge :)